OpenAPI/kiota/install.md

```diff
@@ -62,6 +70,14 @@ Execute the following command to install the tool.
 dotnet tool install --global Microsoft.OpenApi.Kiota
 ```
 
+## Install with asdf
+
+> Disclaimer: The asdf Kiota plugin is maintained and distributed by the community and is not an official Microsoft plugin. Microsoft makes no warranties, express or implied, with respect to the plugin or its use. Use of this plugin is at your own risk. Microsoft shall not be liable for any damages arising out of or in connection with the use of this plugin.
```

Comment:
This text is almost identical to the one above, what do you think about keeping only one of them?

Reply:
because if we share an anchored link, people won't see the disclaimer at the top

Reply:
What do you think about keeping only the note below?

Reply:
meaning only keeping the one under the asdf section and not the one under the community provided options?

Reply:
Correct, given the fact that ASDF is the only community installer at the moment, it should cover the use case

Reply:
I've done an edit pass on the doc and removed the disclaimer at the top. @baywet can you take a look at the changes and let me know if you have any concerns?

Reply:
yes, that works for me, thanks everyone for the help